Vaccination

bolthikaru opened the pull request.

pratikunterwegs committed:
Update noromod_r for vaccination

pratikunterwegs committed:
Add vaccination params

pratikunterwegs committed:
Update Readme with vaccination

pratikunterwegs commented:
This look alright - the tests are failing due to unexpected arguments and non-equivalence between R and C++ but that's exepcted and something that can be fixed today.

bolthikaru committed:
Update noromod_r.R

bolthikaru committed:
Update parameters.R

bolthikaru committed:
Update noromod.cpp

bolthikaru committed:
Update noromod.cpp

pratikunterwegs committed:
Correct noromod_r() - now working

pratikunterwegs committed:
Update noromod Boost odeint implementation, WIP

pratikunterwegs committed:
Update noromod_r() with waning RV2

pratikunterwegs committed:
Correct upsilon in parameters

pratikunterwegs committed:
Fix transitions in unvaccinated compartment

pratikunterwegs committed:
Update noromod cpp with vax 1 stratum

pratikunterwegs committed:
Add second vax stratum noromod_cpp

pratikunterwegs committed:
Enable vaccination and waning for both vax strata

pratikunterwegs committed:
Remove Rcpp-deSolve version

pratikunterwegs committed:
Update Readme

actions-user committed:
Automatic readme update

pratikunterwegs committed:
Re-arrange cols in noromod_r()

pratikunterwegs committed:
Update tests

pratikunterwegs committed:
Annotate Readme

actions-user committed:
Automatic readme update

bolthikaru committed:
Add vaccination start parameter

bolthikaru committed:
Add vaccination start

bolthikaru committed:
Adding seasonal amplitude for over 65

Adding seasonal amplitude term for over 65 to improve fit to surveillance data

bolthikaru committed:
Adding seasonal amplitude for over 65

bolthikaru committed:
Update cpp to include over 65 seasonal forcing term

bolthikaru committed:
Update parameters so default is no vaccination scenario

bolthikaru committed:
Updated normod cpp with vaccination start

Add vaccination start implementation so vaccination starts after the model has reached equilibrium

bolthikaru committed:
Initializing vacc_start

pratikunterwegs commented:
Hey @bolthikaru - how's this looking?

bolthikaru committed:
vaccination model update

Updated so that number of individuals being vaccinated in each age group can be tracked. As those in the S & R compartment are vaccinated, trying to keep track of the numbers vaccinated from each of these groups.
